docs/shared/angular-tutorial/angular-monorepo.md

```diff
@@ -0,0 +1,1032 @@
+
+```json {% fileName="libs/orders/project.json" %}
+{
+  ...
+  "tags": ["type:feature", "scope:orders"]
+}
+```
+
+Then go to the `project.json` of your `products` library and assign the tags `type:feature` and `scope:products` to it.
+
+```json {% fileName="libs/products/project.json" %}
+{
+  ...
+  "tags": ["type:feature", "scope:products"]
+}
+```
+
+Finally, go to the `project.json` of the `shared-ui` library and assign the tags `type:ui` and `scope:shared` to it.
+
+```json {% fileName="libs/shared/ui/project.json" %}
+{
+  ...
+  "tags": ["type:ui", "scope:shared"]
+}
+```
+
+Notice how we assign `scope:shared` to our UI library because it is intended to be used throughout the workspace.
+
+Next, let's come up with a set of rules based on these tags:
+
+- `type:feature` should be able to import from `type:feature` and `type:ui`
+- `type:ui` should only be able to import from `type:ui`
+- `scope:orders` should be able to import from `scope:orders`, `scope:shared` and `scope:products`
+- `scope:products` should be able to import from `scope:products` and `scope:shared`
+
+To enforce the rules, Nx ships with a custom ESLint rule. Open the `.eslintrc.base.json` at the root of the workspace and add the following `depConstraints` in the `@nx/enforce-module-boundaries` rule configuration:
+
+```json {% fileName=".eslintrc.base.json" %}
+{
+  ...
+  "overrides": [
+    {
+      ...
+      "rules": {
+        "@nx/enforce-module-boundaries": [
+          "error",
+          {
+            "enforceBuildableLibDependency": true,
+            "allow": [],
+            "depConstraints": [
+              {
+                "sourceTag": "*",
+                "onlyDependOnLibsWithTags": ["*"]
+              },
+              {
+                "sourceTag": "type:feature",
+                "onlyDependOnLibsWithTags": ["type:feature", "type:ui"]
+              },
+              {
+                "sourceTag": "type:ui",
+                "onlyDependOnLibsWithTags": ["type:ui"]
+              },
+              {
+                "sourceTag": "scope:orders",
+                "onlyDependOnLibsWithTags": [
+                  "scope:orders",
+                  "scope:products",
+                  "scope:shared"
+                ]
+              },
+              {
+                "sourceTag": "scope:products",
+                "onlyDependOnLibsWithTags": ["scope:products", "scope:shared"]
+              },
+              {
+                "sourceTag": "scope:shared",
+                "onlyDependOnLibsWithTags": ["scope:shared"]
+              }
+            ]
+          }
+        ]
+      }
+    },
+    ...
+  ]
+}
+```
+
+To test it, go to your `libs/products/src/lib/products.module.ts` file and import the `OrderListComponent` from the `orders` project:
```

Comment:
I believe the `products.module.ts` is no more there if we generate the library in standalone mode, right?